### DISCUSSION TITLE
Secure Spreadsheets note stuck on "Waiting for Key" - cannot delete it

### DISCUSSION THREAD

**Discussion by eric-pierce** [standardnotes/forum#2478] [General] [answered]
**Describe the bug**
One of my secure spreadsheets notes is stuck on "waiting for key" on all devices except the one it was created on (accessing via web frontend). When I signed out of the one it was created on, I got the error message on that device as well.

This is a recent problem, and previously I was able to access the secure spreadsheet on both the iOS app and the web frontend.

Selecting "move to trash" has no effect, and I can't delete this note.

**To Reproduce**
Unclear

**Expected behavior**
Note accessible everywhere.

**Desktop (please complete the following information):**
 - Browser Firefox, Safari, Chrome, iOS
 - Version web frontend 3.8.13, syncing-server-js backend latest

**Comment by eric-pierce** [comment, marked as answer]
I don't know the cause of this, but I "resolved" it by looking in the edit history to find a version of the spreadsheet I could view, copied the contents to a local excel doc, uninstalled secure spreadsheets, deleted the note (this only worked after uninstalling), restarted all services, re-installed secure spreadsheets, and pasted the contents I had previously copied out.

**Comment by JaspalSuri** [comment]
Hi Eric, sorry to hear that. Thank you for letting us know what you did you did that helped resolve the issue. 

Did you change your account's password at all recently? Which app/browser was the note created on?

Also, (while it might not necessarily related to the issue) may I ask why you're using `syncing-server-js` instead of `Standalone`?

**Comment by eric-pierce** [comment]
@JaspalSuri thanks for the reply! I haven't made any password changes, and the note was created using Firefox.

To your question about sources, I am using the docker-compose setup from Standalone, which leverages the syncing-server-js docker image https://github.com/standardnotes/standalone/blob/main/docker-compose.yml

Here's my docker-compose setup in case you're interested: https://github.com/eric-pierce/Personal-Cloud/blob/main/docker-compose.yml

**Comment by JaspalSuri** [comment]
>I haven't made any password changes, and the note was created using Firefox.

Thanks for those details! I'm sorry to hear that. Hopefully we'll find a way to reliably reproduce this issue. 

>I am using the docker-compose setup from Standalone

Thanks for clarifying; I see what you mean now. 🙂
